ale_linters/json/jq.vim

```diff
@@ -0,0 +1,34 @@
+" Author: jD91mZM2 <me@krake.one>
+
+call ale#Set('json_jq_executable', 'jq')
```

Comment:
@hsanson Note, this variable is shadowing a variable from the formatting counterpart (jq is used for formatting already, where this option is set). I was going to point this out earlier, but forgot - in any case I need help with what the proper way of dealing with this scenario is.

(This is also why I haven't documented this option - there's already docs for an option with the same name)

Reply:
Oh, in this case I would simply call the fixer GetCommand() method within this one or use it directly in the linter definition:

```
function! ale_linters#json#jq#GetCommand(buffer) abort
    return ale#fixers#jq#GetExecutable(a:buffer)
endfunction
```

If  don't think anyone would need to set different binaries for the linter and the fixer. Even if this is the case it can be done using bufffer variables (e.g. b:ale_json_jq_executable).